.github/workflows/build_and_test.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     uses: ./.github/workflows/_build.yml
     with:
       ecr-docker-image-base: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/xla_base
-      gcr-docker-image: gcr.io/tpu-pytorch/xla_base:latest
+      gcr-docker-image: gcr.io/tpu-pytorch/xla_base:cuda_12_0
```

Comment:
Should we upgrade to cuda 12.1 since PyTorch supports 12.1 instead of 12.0?

Reply:
Good idea -- let's do that. Hopefully, 12.0 to 12.1 is straightforward.

Reply:
@vanbasten23 I will handle the actual base image update separately, and probably do it in steps, 12.0 -> 12.1. FYI.